tests/test_language.py

```diff
@@ -78,6 +79,5 @@ def test_eq(self):
     def test_hash(self):
         for name in ["html", "javascript", "json", "python", "rust"]:
             with self.subTest(language=name):
-                lang = Language(getattr(self, name))
-                hash_ = hash(lang) & maxsize << 1
-                self.assertGreater(hash_, 0)
+                lang = Language(cast(object, getattr(self, name)))
+                self.assertGreater(hash(lang) & maxsize << 1, 0)
```

Comment:
>_possible bug_

**✨ AI Suggestion:** The `cast` here seems unnecessary as `Language` constructor accepts a path-like object. Consider removing the cast for better readability.
```suggestion
                lang = Language(getattr(self, name))
                self.assertGreater(hash(lang) & maxsize << 1, 0)
```